metadata_to_json_and_fnamesmap.py

```diff
@@ -158,10 +139,7 @@ def main(args):
         populated_dict = populate_new_dict_with_metadata(metadata, new_dict)
         #print("populated_dict: {}".format(populated_dict))
 
-        dict_with_commons_filenames = add_commons_filenames_to_dict(metadata, populated_dict)
-        #print("dict_with_commons_filenames: {}".format(dict_with_commons_filenames))
-
-        dict_with_smvk_mm_link = add_smvk_mm_link_to_dict(metadata, dict_with_commons_filenames)
+        dict_with_smvk_mm_link =
```

Comment:
I had forgotten to remove this. Removed now since this functionality is moved to create_infotext.py and uses string formatting instead of a preliminary dict as in this case.